src/crh/mod.rs

```diff
@@ -23,3 +25,20 @@ pub trait FixedLengthCRH {
     fn setup<R: Rng>(r: &mut R) -> Result<Self::Parameters, Error>;
     fn evaluate(parameters: &Self::Parameters, input: &[u8]) -> Result<Self::Output, Error>;
 }
+
+pub trait TwoToOneCRH {
+    /// The size for left input and right input
+    const LEFT_INPUT_SIZE_BITS: usize;
+    const RIGHT_INPUT_SIZE_BITS: usize;
+
+    type Output: ToBytes + Clone + Eq + core::fmt::Debug + Hash + Default;
+    type Parameters: Clone + Default;
+
+    fn setup<R: Rng>(r: &mut R) -> Result<Self::Parameters, Error>;
+    fn evaluate(
+        // we do not use `evaluate` because some implementations may use both `CRH` and `TwoToOneCRH` trait, which causes ambiguity
```

Comment:
```suggestion
```

Reply:
thanks for that catch... I forgot to remove the comment. 

I will fix that tomorrow

(I relied too much on the IDE's auto-rename feature. I will manual check the code again tomorrow as well)

Reply:
fixed